src/js/user-profile/reducers/profile.js

```diff
@@ -1,6 +1,7 @@
 import _ from 'lodash/fp';
 
-import { UPDATE_PROFILE_FIELD } from '../actions';
+import { UPDATE_PROFILE_FIELD, PROFILE_LOADING_FINISHED } from '../actions';
+import { UPDATE_LOGGEDIN_STATUS } from '../../login/actions';
 
 // TODO(crew): Romove before this goes to production.
```

Comment:
LOL. I wondered about this long ago but part of me really wanted to know why one would delete the initial state  definition before going to production, and I don't know which crew to ask. Maybe the comment was originally for a different chunk of code that's long since been removed?